learning-labs/modules/local-development/pages/2-run-jenkins.adoc

```diff
@@ -4,21 +4,52 @@ Docker simplifies packaging applications with all their dependencies.
 
 Through Docker, we can have a running Jenkins instance in a matter of seconds. 
 
-In your terminal, run: 
+In your terminal, first build a jenkins image with docker installed. Create a `Dockerfile` with the following:
+
+[source,]
+----
+FROM jenkins/jenkins:lts
+USER root
+RUN apt-get update && apt-get install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/debian \
+   $(lsb_release -cs) \
+   stable"
+RUN apt-get update && apt-get install -y docker-ce docker-ce-cli containerd.io 
+EXPOSE 8080
+----
+
+From the same directory as your Dockerfile, build the image:
+[source,]
+----
+docker build -t jenkins:lts-docker .
+----
+
+[NOTE]
+====
+Running Jenkins as a docker container on a BAH managed machine requires specific certificates and tooling to be installed. A Dockerfile meeting these requirements can be built from the https://github.boozallencsn.com/solutions-delivery-platform/bah-jenkins[solutions-delivery-platform/bah-jenkins] repo.
+====
+
+Then, to start jenkins, run:
 
 (If running Docker for Windows, please refer to Important section below)
 
 [source,]
 ----
     docker run --name jenkins \
-    -v $(which docker):/usr/local/bin/docker \
     -v /var/run/docker.sock:/var/run/docker.sock \
     --privileged \
     --user root \
     -p 50000:50000 \
     -p 8080:8080 \
     -d \
-    jenkins/jenkins:lts
+    jenkins:lts-docker
```

Comment:
the documentation on line 84 needs to reflect this new name of `jenkins:lts-docker`

Reply:
that is the only isssue.  I will approve and merge after that. thanks for finding the issue and submitting the PR.

Reply:
good catch, sorry about that! line 84 updated now.